data/json/vehicles/trucks.json

```diff
@@ -891,10 +891,10 @@
       { "x": -2, "y": 1, "chance": 20, "items": [ "blanket" ] },
       { "x": -3, "y": 0, "chance": 20, "items": [ "blanket" ] },
       { "x": -3, "y": 1, "chance": 20, "items": [ "blanket" ] },
-      { "x": -4, "y": 0, "chance": 10, "items": { "item": "dog_food", "container-item": "can_medium" } },
-      { "x": -4, "y": 0, "chance": 10, "items": { "item": "dog_food", "container-item": "can_medium" } },
-      { "x": -4, "y": 1, "chance": 10, "items": { "item": "cat_food", "container-item": "can_food" } },
-      { "x": -4, "y": 1, "chance": 10, "items": { "item": "cat_food", "container-item": "can_food" } },
+      { "x": -4, "y": 0, "chance": 10, "//": { "item": "dog_food", "container-item": "can_medium" } },
```

Comment:
Why comment instead of items here and in three lines below?

Reply:
Because spawning items via an elaborated  syntax similar to item group is not implemented for vehicles. This system allows simple arrays of strings (item type ids), or a just a single string, but nothing else.

Alternatively one has to create an explicit item group that allows container-items.

Reply:
Right. I thought it was a mapgen.